client/performance-profiler/index.web.ts

```diff
@@ -8,6 +9,23 @@ import {
 } from './controller';
 
 export default function () {
+	const lang = getAnyLanguageRouteParam();
+
+	page(
+		`/${ lang }/speed-test-tool/`,
+		PerformanceProfilerDashboardContext,
+		makeLayout,
+		clientRender
+	);
+	page( `/${ lang }/speed-test-tool/weekly-report`, WeeklyReportContext, makeLayout, clientRender );
+	page(
+		`/${ lang }/speed-test-tool/weekly-report/unsubscribe`,
+		WeeklyReportUnsubscribeContext,
+		makeLayout,
+		clientRender
+	);
+	page( `/${ lang }/speed-test-tool*`, notFound, makeLayout, clientRender );
+
 	page( '/speed-test-tool/', PerformanceProfilerDashboardContext, makeLayout, clientRender );
```

Comment:
The non language-specific routes should be redundant now?

Reply:
Fixed at https://github.com/Automattic/wp-calypso/pull/95001/commits/8b27404898442d4cdc91bf57d80a3e706bba0edd